recipes/poethepoet/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "poethepoet" %}
+{% set version = "0.10.0" %}
+{% set python_min = "3.6" %}
```

Comment:
Conda-forge supports Python 3.9 and above only. No need to add this variable unless your min Python is higher than 3.9.

```suggestion
```